tools/check-API-refs.py

```diff
@@ -9,7 +9,7 @@
 root = dirname(dirname(abspath(__file__)))
 patsy_ref = root + "/doc/API-reference.rst"
 
-doc_re = re.compile("^\.\. (.*):: ([^\(]*)")
```

Comment:
Better to make it a raw string, `r"..."`

Reply:
Changed.